src/main/java/Duke.java

```diff
@@ -1,55 +1,94 @@
+import java.util.Arrays;
 import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class Duke {
-    public static void main(String[] args) {
+    private static void welcomeMessage() {
         String greetings = "Hello! I'm Duke\n"
                 + "What can I do for you?\n";
         System.out.println(greetings);
+    }
+
+    private static void goodbyeMessage() {
+        String goodbyeMessage = "Bye. Hope to see you again soon!";
+        System.out.println(goodbyeMessage);
+    }
+
+    public static void addTaskToTaskList(String userInput, int taskNumber, Task[] taskList) {
+        String[] userInputSplitArray = userInput.split(" ");
+        String taskType = userInputSplitArray[0];
+
+        String taskDescription = "";
```

Comment:
maybe you can add some comment in this section of code because it is not that easy to understand the mechanism here.